app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,26 @@
+class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  def google_oauth2
+    @user = User.from_omniauth(request.env["omniauth.auth"])
+
+    if @user.persisted?
+      flash[:notice] = "devise.omniauth_callbacks.success", :kind => "Google"
```

Comment:
That line was straight from the omniauth-google-oauth2 github page. 
`flash[:notice] = I18n.t "devise.omniauth_callbacks.success", :kind => "Google"`
https://github.com/zquestz/omniauth-google-oauth2

I removed the I18n.t part since I wasn't sure if our app supports Internationalization. From what I understand, I18n.t would be used here if we wanted to provide a translation for the success message. I could be wrong though, I've never used this in past apps. 

I can certainly add it back to that line.